tasks/wkhtmltopdf.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Check if wkhtmltopdf was install
+  stat:
+    path: "/usr/bin/wkhtmltopdf"
+  register: wkhtmltopdf_was_installed
+
+- name: Check wkhtmltopdf version
+  command: "/usr/bin/wkhtmltopdf --version"
+  register: wkhtmltopdf_version
+  when: "wkhtmltopdf_was_installed.stat.exists"
+  changed_when: False
+  failed_when: False
+
+- name: Remove the wktmltopdf packaged installed to install the correct version
+  become: yes
+  apt:
+    pkg: wkhtmltopdf
+    state: absent
+  register: install_wkhtmltopdf
+  when: "widgetizer_target_version is not defined or 0.12.4 not in wkhtmltopdf_version.stdout"
+
+- name: Download wkhtmltopdf version 0.12.4
```

Comment:
We can get the `.deb` package from: https://wkhtmltopdf.org/downloads.html using the checksum